### PR TITLE
Alphabetize and consistency in "Directory structure" and "Categories"…

### PR DESCRIPTION
**Pull request by rchekaluk**
Cleanup README:
* "Directory structure" section: alphabetize
* "Categories" section: make consistent with actual directory structure and alphabetize

### CODE REVIEW

**Comment by Phantsure** [comment]
@rchekaluk reorder is fine. Please don't remove categories.

File: README.md [hunk 1]
```diff
@@ -31,14 +31,15 @@ For example: `ci/django.yml` and `ci/properties/django.properties.json`.
 * `categories`: the categories that it will be shown under. Choose at least one category from the list [here](#categories). Further, choose the categories from the list of languages available [here](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml). When a user views the available templates, those templates that match the same language will feature more prominently.
 
 ### Categories
-* continuous-integration
-* deployment
-* testing
+* automation
+* ci
 * code-quality
-* code-review
```
Comment (Phantsure) [inline README.md:38]: let this stay

Comment (rchekaluk) [reply README.md:38]: Done

File: README.md [hunk 2]
```diff
@@ -31,14 +31,16 @@ For example: `ci/django.yml` and `ci/properties/django.properties.json`.
 * `categories`: the categories that it will be shown under. Choose at least one category from the list [here](#categories). Further, choose the categories from the list of languages available [here](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml). When a user views the available templates, those templates that match the same language will feature more prominently.
 
 ### Categories
-* continuous-integration
-* deployment
-* testing
+* automation
+* ci
```
Comment (Phantsure) [inline README.md:35]: This can be removed as we have `Continuous-Integration`

File: README.md [hunk 3]
```diff
@@ -31,14 +31,16 @@ For example: `ci/django.yml` and `ci/properties/django.properties.json`.
...
 * code-quality
 * code-review
+* code-scanning
+* continuous-integration
 * dependency-management
+* deployments
```
Comment (Phantsure) [inline README.md:41]:
```suggestion
* deployment
```

File: README.md (hunk 2 above)
Comment (rchekaluk) [reply README.md:35]: I was suggesting to make "Categories" consistent with "Directory structure". This repo has directories `automation` and `ci`, so IMO these Categories should remain.

File: README.md (hunk 3 above)
Comment (rchekaluk) [reply README.md:41]: The repo contains directory `deployments` not `deployment`

Comment (Phantsure) [reply README.md:41]: Yes but the category is `deployment`. I also just got this difference 🤔

File: README.md (hunk 2 above)
Comment (Phantsure) [reply README.md:35]: directory structure is used while showing the templates. So it is coupled. Category is also used and it not same as directory names

Comment (Phantsure) [reply README.md:35]: I am sorry. In that case I have to reject. Thanks for your contribution

**Pull request closed without merging**